### PR TITLE
fix document reference link

### PR DESCRIPTION
**Pull request by jaypume**
Please have a preview on [this page](https://ianvs1.readthedocs.io/).

### CODE REVIEW

File: docs/Makefile [hunk 1]
```diff
@@ -18,3 +18,15 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+clean:
+	rm -rf "$(BUILDDIR)"
+
+api:
+	rm -rf ./source/api/*
+	@$(SPHINXAPIDOC) -M -o ./lib-api/joint_inference ../lib/sedna/joint_inference
```
Comment (MooreZheng) [inline docs/Makefile:27]: Sedna -> Ianvs? Ianvs does not contain api though, might remove this statement.

File: docs/Makefile [hunk 2]
```diff
@@ -18,3 +18,15 @@ help:
 # "make mode" option.  $(O) is meant as a shortcut for $(SPHINXOPTS).
```
Comment (MooreZheng) [inline docs/Makefile:18]: Comment is needed to avoid misuse.

File: docs/conf.py [hunk 3]
```diff
@@ -6,27 +6,43 @@
```
Comment (MooreZheng) [inline docs/conf.py:6]: Comment is needed to avoid misuse.

File: docs/Makefile [hunk 4]
```diff
@@ -18,3 +18,15 @@ help:
...
 %: Makefile
 	@$(SPHINXBUILD) -M $@ "$(SOURCEDIR)" "$(BUILDDIR)" $(SPHINXOPTS) $(O)
+
+clean:
+	rm -rf "$(BUILDDIR)"
```
Comment (JimmyYang20) [inline docs/Makefile:23]: Check whether the path exists.

**Pull request merged**